brainscore_language/plugins/sg_tse/data.py

```diff
@@ -0,0 +1,16 @@
+class user_def_params():
+    def syntaxgymtestsuite():
+        testsuitepath = 'test_suite.json'
+        return testsuitepath
+
+    def bench_id():
+        benchid = 'SG-TSE'
+        return benchid
+
+    def model_id():
+        modelid = 'gpt2'
+        return modelid
+
+    def metric_id():
+        metricid = 'accuracy'
+        return metricid
```

Comment:
this file should go away and make its way into benchmark instantiations

Reply:
I removed the user_def_params class and all references to it.  I also removed the data.py file.